bin/logstash.bat

```diff
@@ -47,5 +50,34 @@ set JAVA_OPTS=%LS_JAVA_OPTS%
 
 rem jruby launcher will pickup JAVA_OPTS set above to set the JVM options before launching jruby
 %JRUBY_BIN% "%LS_HOME%\lib\bootstrap\environment.rb" "logstash\runner.rb" %*
+goto end
+
+:version
+rem make the script work without LS_HOME variable set
+if "%LS_HOME%" == "" (
+	for %%a in ("%~dp0..") do set "LS_HOME=%%~fa"
+)
+
```

Comment:
this `LS_HOME` handling is no required since the first thing this bat file does is to call `setup.bat` which handle the setup of `LS_HOME` see https://github.com/elastic/logstash/blob/40827a53eb6767dc019c926bd5e5bb5d81c1f246/bin/setup.bat#L5-L19